includes/Modules/Analytics/Settings.php

```diff
@@ -127,6 +131,21 @@ function ( $option ) {
 		);
 	}
 
+	/**
+	 * Returns keys for owned settings.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array An array of keys for owned settings.
+	 */
+	public function get_owned_keys() {
+		return array(
+			'accountID',
+			'propertyID',
+			'profileID',
```

Comment:
Nit-picking, this wasn't part of the ACs and technically shouldn't be a problem since it's tied to `propertyID`, but we probably should also include `internalWebPropertyID` here.

Reply:
Added.